README.md

```diff
@@ -1,17 +1,21 @@
 Chitter Challenge
 =================
 
-* Feel free to use Google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
+# How to use database
```

Comment:
good readme instructions

Reply:
could have added instructions on running tests and using the project eg run bundle, run rackup